### DISCUSSION TITLE
Running Pekko on Scala 2.11

### DISCUSSION THREAD

**Discussion by yushah8892** [apache/pekko#1281] [Q&A] [answered]
I have project that uses scala 2.11 and earlier I used Akka framework for concurrnecy usecases. Now I want to use Pekko as replacement of akka.  Migrating existing project to scala > 2.12 is going to be a huge task for me. 
So, How can I work with pekko in my project that uses scala 2.11 ?

**Comment by He-Pin** [comment, marked as answer]
I think most code you can compile with 2.11 should be ok with 2.12 too, and at least for me, I have no interest in maintaining a branch/fork for 2.11.x, how about you first try to upgrade your project to 2.12.x?

**Reply by pjfanning** [reply]
This is very unlikely to happen. Scala 2.11 is no longer getting patched. Community members may be able help you upgrade to Scala 2.12 or above if you provide some details of what is blocking you.
